crates/vfio-ioctls/src/vfio_device.rs

```diff
@@ -877,6 +877,11 @@ impl VfioDevice {
         self.irqs.get(&irq_index)
     }
 
+    /// Get information about all VFIO IRQs.
```

Comment:
Do we need to add some unit tests for the new API?

Reply:
ut is added.